templates/quickstart-jira-dc.template.yaml

```diff
@@ -1002,6 +1005,13 @@ Resources:
         - Key: Cluster
           Value: !Ref AWS::StackName
           PropagateAtLaunch: true
+        # NOTE: The leading COMMIT/TIMESTAMP are used to locate the position to update; see scripts/update-tags.py
```

Comment:
We don't have the script to update the commit/timestamp in this repo. Should we update the comment to say use the script from Atlassian services or should we duplicate the script?